src/WebUI/Resources.js

```diff
@@ -66,5 +61,14 @@ define("Environments/Providers/Kubernetes/Resources", ["require", "exports"], fu
     exports.WorkloadText = "Workload";
     exports.WorkloadDetails = "{0} details";
     exports.PodStatusPending = "Pending";
-    exports.MoreImagesText = "{0} and {1} more"
+    exports.MoreImagesText = "{0} and {1} more";
+    exports.LearnMoreKubeResourceText = "Learn more about Kubernetes resource";
+    exports.DeployKubeResourceText = "Use this resource to deploy workloads and services";
+    exports.StartUsingKubeResourceText = "Start using resource!";
+    exports.StartingUsingServiceText = "You can also deploy services and track them in here";
+    exports.DeployServices = "Deploy services!";
+    exports.DeployWorkloads = "Deploy workloads!";
+    exports.WorkloadsZeroDataText = "You can also deploy all kinds of workloads like deployment, replicaset, daemonset etc., and track them in here";

```

Comment:
would you able to have an image for this flow?

Reply:
@madhuv-msft Please refer to https://user-images.githubusercontent.com/45386516/53316435-e57ba100-38ed-11e9-8f38-12633ed46e2b.jpg. Screen 2